guides/review.rst

```diff
@@ -101,17 +101,11 @@ src then it counts.
 3. Hypothesis must never be *flaky*. Flakiness here is
    defined as anything where a test fails and this does not indicate
    a bug in Hypothesis or in the way the user wrote the code or the test.
-4. The version number must be kept up to date, following
-   `Semantic Versioning <http://semver.org/>`_ conventions: The third (patch)
-   number increases for things that don't change public facing functionality,
-   the second (minor) for things that do but are backwards compatible, and
-   the first (major) changes for things that aren't backwards compatible.
-   See the section on API changes for the latter two.
-5. The changelog should be kept up to date by creating a RELEASE.rst file in
-   the root of the repository. Make sure you build the documentation and
-   manually inspect the resulting changelog to see that it looks good - there
-   are a lot of syntax mistakes possible in RST that don't result in a
-   compilation error.
+4. The changelog must be kept up to date by creating a RELEASE.rst file in
+   the root of the repository.  Notes on whether to bump the minor version
+   or patch version can be found in ``CONTRIBUTING.rst``.  Make sure you
```

Comment:
Just to make sure I'm understanding this right... we're asking reviewers to check out the branch under review, build the docs, and look at the changelog?

It seems to me like maybe it should be sufficient for 1 person to do that: either the author is someone we trust and they say they did it, or the reviewer should.

Reply:
I have to admit, I think this bit of the guide was more aspirational than practical. I'm not sure we ever do that in practice, and the guide should probably be updated to reflect that. :-(

The problem with this is that it's really easy to forget to check, and even though we validate the RST, RST is one of those horrible formats where invalid syntax usually results in wrong output rather than a compile error.

Ideally what we would do is update the build to upload built docs to a time limited S3 bucket or something so it was easy to take a look at. But oh god yet more elaborate build engineering.